rel/overlay/etc/default.ini

```diff
@@ -187,6 +188,14 @@ credentials = false
 ; List of accepted methods
 ; methods =
 
+[x_frame_options]
+; Settings same-origin will return X-Frame-Options: SAMEORIGIN.
+; If same origin is set, it will ignore the hosts setting
+; same_origin = true
+; Settings hosts will return X-Frame-Options: ALLOW-FROM https://example.com/
+; List of hosts separated by a comma. * means accept all
+; hosts =
```

Comment:
what are the consequences of having typos here besides the embedder not being able to put the site in frames and complaining to us?